src/lib.rs

```diff
@@ -192,6 +180,15 @@ pub enum Spin {
     CCW(Direction),
 }
 
+impl Into<Direction> for Spin {
```

Comment:
Why not `From<Spin> for `Direction`? https://doc.rust-lang.org/std/convert/trait.Into.html "Library authors should not directly implement this trait, but should prefer implementing the From trait (...)"?

Reply:
Thank you, my bad, will revise